library/MedEx/API.php

```diff
@@ -0,0 +1,2092 @@
+                                <li id="menu_pend_recalls" name="menu_pend_recalls"> <a  onclick="goReminderRecall('Recalls');" id="BUTTON_pend_recalls_menu" href="#"> <?php echo xlt("Recall Board"); ?></a></li>
+                            <?php if ($logged_in) { ?>
+                                 <li class="divider"><hr /></li>
+                                <li id="menu_pend_recalls" name="menu_pend_recalls"> <a href='https://medexbank.com/cart/upload/index.php?route=information/campaigns' class='nowrap left' target='_blank' id="BUTTON_pend_recalls_menu"> <?php echo xlt("Recall Campaigns"); ?></a></li>
+                            <?php } ?>
+                            </ul>
+                        </li>
+                    </ul>
+                </div><!-- /.navbar-collapse -->
+            </div>
+        </nav>
+        <?php 
+        $error=$this->MedEx->getLastError();
+        if (!empty($error['ip'])) { ?>
+        <div class="alert alert-danger" style="width:50%;margin:30px auto 5px;font-size:0.9em;text-align:center;">    
+            <?php  echo $error['ip']; ?>
+        </div>
+        <?php   }
+    }
+    public function preferences($prefs='') {
+        if (empty($prefs)) {
+            $prefs = sqlFetchArray(sqlStatement("SELECT * from medex_prefs"));
+        }
+        ?>
+        <div class="row">
+            <div class="col-sm-12">
+                <div class="showRecalls" id="show_recalls" style="text-align:center;margin:40px auto;">
+                    <div class="title"><?php echo xlt('Reminder/Recall Preferences'); ?></div>
+                    <div name="div_response" id="div_response"><br /></div>
+                        <form action="#" name="save_prefs" id="save_prefs">
+                            <div class="row">
+                                <div class="col-sm-1"><input type="hidden" name="go" id="go" value="Preferences">
+                                </div>
+                                <div class="col-sm-10 div-center" id="daform2"><div class="divTable2">
+                                        <div class="divTableBody prefs">
+                                            <div class="divTableRow">
+                                                <div class="divTableCell divTableHeading"><?php echo xlt('General'); ?></div>
+                                                <div class="divTableCell indent20">
+                                                    <input type="checkbox" class="update" name="ME_hipaa_default_override" id="ME_hipaa_default_override" value="1" <?php 
+                                                        if ($prefs['ME_hipaa_default_override']=='1') echo 'checked ="checked"'; ?>/>
+                                                    <label for="ME_hipaa_default_override" class="input-helper input-helper--checkbox" title='<?php echo xla('Default: "checked".
```

Comment:
translations can's have newlines in them. Either make it all one line or break it up (making it one line would be best).